packages/shared/useTimeoutFn/index.ts

```diff
@@ -10,11 +10,12 @@ import { isClient } from '../utils'
  * @param immediate
  */
 export function useTimeoutFn(
-  cb: () => any,
+  cb: (...args: unknown[]) => any,
   interval?: number,
   immediate = true,
 ) {
   const isActive = ref(false)
+  const isPending = ref(false)
```

Comment:
What is this for?

Reply:
sometimes i want to know it  whether in pending and i think the isActive is useless. maybe u could intead isPending of isActive
for example
```vue
<template>
  <button :disabled="isPending">
    <slot />
    <Spinner v-if="isPending" />
  </button>
</template>
```

Reply:
I see, but looks like you forget to export it? Yeah, let's remove `isActive` in favor of `isPending`.